### ISSUE TITLE
Check DNSSEC

### ISSUE THREAD

**Issue by kazet** [CERT-Polska/mailgoose#80]

**Comment by btsouth** [comment]
Hi! I'd like to take this one. One design question first, since how DNSSEC is surfaced affects existing results.

`checkdmarc` (already pinned at 5.17.1) exposes `checkdmarc.dnssec.test_dnssec(domain, *, nameservers=..., timeout=...) -> bool`, so this needs no new dependency and matches how the SPF/DMARC checks already reuse `checkdmarc`. I'd call it from `scan_domain()` in `scan/libmailgoose/scan.py` with the same `nameservers`/`timeout`.

**How would you like DNSSEC surfaced?**

1. **Advisory warning (my recommendation):** report status as informational, e.g. a warning when DNSSEC isn't enabled, without touching `num_checked_mechanisms`. Keeps the existing SPF/DMARC/DKIM scoring intact so currently-passing domains don't suddenly look incomplete.
2. **Full scored mechanism:** add a `DNSSECScanResult`, include it in `mechanisms`, bump the count to 4, render its own card. More visible, but changes every existing result's denominator.

Either is fine, I just want to match your intent before writing it. I'll handle lookup failures gracefully (treat as "could not verify", not a hard error) and run any new strings through `{% trans %}` + `scripts/update_translation_files`.

Let me know which option you prefer and I'll open a PR.

**Comment by mimi89999** [comment]
The vast majority of domains don't support DNSSEC and some TLDs don't support it. I don't think that it makes much sense to warn about that.

**Comment by btsouth** [comment]
Good point, I don't want to add noise either. Since this issue was opened by kazet, I'll leave the call to the two of you. One option that avoids flagging the many domains without DNSSEC: show it as neutral informational status, or only a positive note when it is enabled, rather than a warning. Happy to take that route or to drop it, whichever you prefer.

**Comment by kazet** [comment]
If adding DNSSEC check would cause too much spam in the alerts, let's skip it.
